src/query/execute_schema.c

```diff
@@ -2625,7 +2625,7 @@ do_rename (PARSER_CONTEXT * parser, PT_NODE * statement)
       const char *old_qualifier_name = pt_get_qualifier_name (parser, current_rename->info.rename.old_name);
       const char *new_qualifier_name = pt_get_qualifier_name (parser, current_rename->info.rename.new_name);
 
-      if (pt_str_compare (old_qualifier_name, new_qualifier_name, CASE_INSENSITIVE) != 0)
+      if (old_qualifier_name && new_qualifier_name && pt_str_compare (old_qualifier_name, new_qualifier_name, CASE_INSENSITIVE) != 0)
```

Comment:
How about modifying it like below?
`if (old_qualifier_name && new_qualifier_name && intl_identifier_casecmp(old_qualifier_name, new_qualifier_name) != 0)`

But if any of A and B are null, is it correct to judge that it is normal without having to compare? Isn't that an error?

Reply:
If it is a system class, new_qualifier_name is null because unique_name does not contain an owner.
So the error occurs because the owner is different.

The error is not appropriate, and it has been changed to match the existing test case answer.

I would change to intl_identifier_casecmp() instead of pt_str_compare() function.